cmd/policy/policy.go

```diff
@@ -223,6 +223,9 @@ func testCommand() *cobra.Command {
 			if res := m.ValidatePolicies(); res.Errors != nil {
 				return res.Errors
 			}
+			if cmd.Flags().Lookup("strict-loading").Changed {
+				m.StrictLoading = true
+			}
```

Comment:
I dont think this is working how you intend.
The way the flag is set up will set the value to true just by adding the `--strict-loading` flag to the command.
I was able to add the flag to the `tf-scan opal` scan and it applied the flag to the generated opal run command.

Reply:
I intentionally wanted the flag's presence be implicitly true i.e. just giving `--strict-loading` should indicate that it is true.
But I see that the flag does not work correctly for `--strict-loading=false` since I'm purely working off the flag's presence/absence - will fix.

Reply:
@ellielacework I simply removed this code so this is now a plain boolean flag with default `false`

`policy test` (flag not given) --> strict loading is false
`policy test --strict-loading` (value not given) --> strict loading is true
`policy test --strict-loading=true` --> strict loading is true
`policy test --strict-loading=false` --> strict loading is false

Reply:
Is it supposed to be restricted to the `policy test` command only?
Because I can run it with a regular scan, for example `tf-scan -d <dir> --strict-loading`

Reply:
There's value in having it for actual scans because glossing over file load errors is bad there too.
It needs careful testing but it might turn out that we should enable it by default for both tests and scans.

I could try to scope the option so it only gets included for opal, because I imagine we'll never implement it for other tools.

Reply:
Took another look @ellielacework and it seems like runOpts is the right place for this option; given that `soluble tf-scan` and `soluble policy` are separate commands; and the latter does not use toolOpts